src/components/ProjectPreview.jsx

```diff
@@ -24,13 +25,13 @@ export default function ProjectPreview({
     >
       <div className="project-preview__label">
         {preview.slice(0, MAX_LENGTH)}
-        {project.updatedAt && (
+        {!isNil(project.updatedAt) && (
           <div className="project-preview__timestamp">
             {moment(project.updatedAt).fromNow()}
           </div>
         )}
       </div>
-      {(function showArchived() {
+      {(() => {
```

Comment:
The linter didn't like unnamed function declarations but was fine with the arrow syntax